src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.xaml

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Rule Name="AdditionalFiles"
+      PageTemplate="generic"
+      PropertyPagesHidden="true"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+  <Rule.DataSource>
+    <DataSource HasConfigurationCondition="false"
+                ItemType="AdditionalFiles"
+                Persistence="ProjectFile"
+                SourceOfDefaultValue="AfterContext" />
+  </Rule.DataSource>
+
+  <BoolProperty Name="ExcludeFromCurrentConfiguration"
```

Comment:
My initial assumption was I could remove most of this from the file after this point, but I see all of these also exist in the "None.xaml" rule which makes me think code generally is expecting these?

Reply:
Unfortunately the _why_ of many of these properties has been lost to time. Please drop everything other than "ExcludeFromCurrentConfiguration" (the explanation is long so I'll skip it unless you ask 😄) and "Link" (since that is the motivation for all this), and let's just plan on adding more back in if needed.

Reply:
Sounds good, will do!

Reply:
As we later observed: there are tests that assert that the properties here exist and match what's in the None rule, so we're instead following that test.